monai/transforms/transforms.py

```diff
@@ -139,7 +139,9 @@ def __call__(self, data_array, original_affine=None, original_axcodes=None):
 @export
 class LoadNifti:
     """
-    Load Nifti format file from provided path.
+    Load Nifti format file or files from provided path.
+    If load a list of files, stack them together and add a new dimension as first dim, and use the meta data of
```

Comment:
Load Nifti format file or files from provided path. If loading a list of files, stack them together and add a new 
dimension as first dimension, and use the meta data of the first image to represent the stacked result. Note that 
the affine transform of all the images should be same.